multiload/org.mate.panel.applet.multiload.gschema.xml.in

```diff
@@ -61,11 +61,11 @@
       <default>'#00e675'</default>
       <summary>Graph color for shared memory</summary>
     </key>
-    <key name="memload-color2" type="s">
+    <key name="memload-color3" type="s">
```

Comment:
....and here you renamed the Key `memload-color2`.
In result the key is lost and the applet segfault.

Reply:
And this is a new one!